### ISSUE TITLE
UOC BC data load

### ISSUE THREAD

**Issue by zperova** [PDXFinder/pdxfinder#176]
### Description:

Load data to dev and update samples according to latest communication. Minutes [here](https://docs.google.com/document/d/1wNVvouX7JCk92N4ZJuIDVt7L4jFpRCckNj87NjsqE10/edit)
Jira ticket [PDXI-828](https://www.ebi.ac.uk/panda/jira/browse/PDXI-828?workflowName=jira&stepId=5)

### Acceptance criteria:

- [x] necessary models removed
- [x] updated data loaded
- [ ] patient treatment are mapped to ontologies

**Comment by zperova** [comment]
CNA missing symbols fixed

**Comment by zperova** [comment]
rewrite @zperova

**Comment by Afollet** [comment]
Reviewing documentation today.

**Comment by Afollet** [comment]
@zperova patient treatments need to be mapped to ontologies via the admin interface for UOC-BC

**Comment by Afollet** [comment]
Martin has more data, but it is waiting on publication. Therefore, it may be sometime until the PDX Finder needs to ingest more data from UOC-BC.

**Comment by zperova** [comment]
Thanks @Afollet I will make a note of it for the future.

**Comment by Afollet** [comment]
This ticket contains the requirement patient treatment data is mapped. Should we reopen?

I would also like to re-annotate the mutation data for this set. However, I view that as a low priority task that can go into the back log since the mut data was previously release as is.

**Comment by zperova** [comment]
@Afollet good point! thanks for noting.

**Comment by zperova** [comment]
New ticket for mappings: https://github.com/PDXFinder/pdxfinder/issues/320
closing this one
